spec/lib/hyrax/collections_migration_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
```

Comment:
Minor detail, fyi: because the require is already done in `.rspec`, this line is unnecessary here.